Sources/WalletConnectRelay/RelayURLFactory.swift

```diff
@@ -15,10 +15,10 @@ struct RelayUrlFactory {
         self.socketAuthenticator = socketAuthenticator
     }
 
-    func create() -> URL {
+    func create(fallback: Bool) -> URL {
         var components = URLComponents()
         components.scheme = "wss"
-        components.host = relayHost
+        components.host = fallback ? NetworkConstants.fallbackUrl : relayHost
```

Comment:
does this mean that if a wallet wants to enable fallback, for all it's users regardless of their location the url will be `.org` for all of them?